cmd/2019_5/main.go

```diff
@@ -16,22 +16,33 @@
 
 func main() {
 	const (
-		inputAC      = 1
-		inputTR      = 5
-		resComputeAC = 9938601
-		resComputeTR = 4283952
-		separator    = ","
+		inputAC       = 1
+		inputTR       = 5
+		outputCountAC = 10
+		resComputeAC  = 9938601
+		resComputeTR  = 4283952
+		separator     = ","
 	)
 
 	flag.Parse()
 
 	var (
-		instructionsAC = read.Read(flagInput, separator)
-		instructionsTR = make([]int, len(instructionsAC))
+		instructionsAC       = read.Read(flagInput, separator)
+		inChanAC, inChanTR   = make(chan int, 1), make(chan int, 1)
+		outChanAC, outChanTR = make(chan int, outputCountAC), make(chan int, 1)
+		instructionsTR       = make([]int, len(instructionsAC))
 	)
 
 	copy(instructionsTR, instructionsAC)
 
-	utils.AssertEqual(intcode.Compute(instructionsAC, intcode.Output, inputAC), resComputeAC)
-	utils.AssertEqual(intcode.Compute(instructionsTR, intcode.Equals, inputTR), resComputeTR)
+	inChanAC <- inputAC
+	intcode.Compute(instructionsAC, intcode.Output, inChanAC, outChanAC)
+	for i := 1; i < outputCountAC; i++ {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '1', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)